### DISCUSSION TITLE
Extra characters "ff" appear when pressing ":" in normal mode

### DISCUSSION THREAD

**Discussion by fdelacruz** [LazyVim/LazyVim#2258] [Q&A]
Anybody experiencing this?
Since the latest version of neovim v0.10.0-dev extra characters "ff" appear where the cursor stood when pressing ":" in normal mode.
<img width="682" alt="Screen Shot 2023-12-16 at 12 55 44 PM" src="https://github.com/LazyVim/LazyVim/assets/148098/c6d76ff1-ab7c-46ef-bf7f-76fb8603b38f">
It only happens when noice is enabled..

**Comment by dpetka2001** [comment]
This is not default LazyVim behavior. I just tried with a fresh LazyVim starter template and it doesn't show like this.
Must be something in your personal configuration. I tested with Neovim `ec74538` commit which is the latest.

**Reply by fdelacruz** [reply]
Well, do you have noice enabled?

**Reply by dpetka2001** [reply]
`noice` is enabled by default in the LazyVim starter template. I didn't use my own configuration. I cloned a new LazyVim starter and tested with that.

**Reply by dpetka2001** [reply]
Can you reproduce with a new LazyVim starter? Follow the instructions in this [comment](https://github.com/LazyVim/LazyVim/issues/2131#issuecomment-1841402320) to not mess with your own personal configuration.

**Reply by fdelacruz** [reply]
The characters still show up.
<img width="570" alt="Screen Shot 2023-12-27 at 7 05 30 PM" src="https://github.com/LazyVim/LazyVim/assets/148098/37ea1ee3-b5da-4ee8-afca-0e79fc6c35f2">

Like i said, it doesn't happen if I disable noice..

**Reply by dpetka2001** [reply]
I can't reproduce this with a new LazyVim starter, neither in my personal configuration and I've never seen something like this either. 

Maybe it has something to do with your terminal? Could you try with kitty or wezterm to see if they behave the same?

**Reply by fdelacruz** [reply]
I just using iTerm2 and it never happened before using vanilla neovim nor lunarvim.

**Reply by dpetka2001** [reply]
Try with kitty or wezterm to have a comparison.

**Reply by fdelacruz** [reply]
I doesn't do it in wezterm

**Reply by dpetka2001** [reply]
So it's some kind of incompatibility between `noice.nvim` and iterm2. Maybe raise an issue at `noice` repo and hope that the maintainer will be able to find a solution? I don't use Mac, so I'm not able to reproduce this issue.

**Reply by fdelacruz** [reply]
I did open an issue at `noice`repo two weeks ago, no reply yet.
